pkg/config/deployment_ecs.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+// EcsDeploymentSpec represents a deployment configuration for ECS application.
+type EcsDeploymentSpec struct {
+	GenericDeploymentSpec
+	// Input for ECS deployment such as where to fetch source code...
+	Input EcsDeploymentInput `json:"input"`
+	// Configuration for quick sync.
+	QuickSync EcsSyncStageOptions `json:"quickSync"`
+}
+
+// Validate returns an error if any wrong configuration value was found.
+func (s *EcsDeploymentSpec) Validate() error {
+	if err := s.GenericDeploymentSpec.Validate(); err != nil {
+		return err
+	}
+	return nil
+}
+
+type EcsDeploymentInput struct {
+	// The name of service definition file placing in application directory.
+	// Default is servicedef.yaml
+	ServiceDefinition string `json:"serviceDefinition"`
```

Comment:
`ServiceDefinitionFile` and `TaskDefinitionFile` for better understand.

Reply:
I added a suffix to them.